sdc_visualization/server.py

```diff
@@ -458,12 +481,87 @@ def dataset_slice():
     ds.close()
     return jsonify(collection)
 
+@blueprint.route('/api/get_profiles', methods=['GET', 'POST']) # rename to profile as it is not timeseries
+@cross_origin()
+def get_profiles():
+    """ Return profile for selected points"""
+    # read inputs
+    cdi_ids_input = request.args.getlist("cdi_ids")
+    
+    dataset = request.values.get("dataset")
+
+    ds = get_ds(dataset=dataset)
+    if ds is None:
+        return jsonify({
+            'error': 'data not loaded'
+        })
+    # get the variable that has the cdi_ids
+    cdi_id_var = get_cdi_id_var(ds = ds)
+
+    # create a list with all the cdi_ids
+    cdi_ids = netCDF4.chartostring(ds.variables[cdi_id_var][:]) 
+
+    # create a list with the idxs of the given cdi_ids
+    idxs = []
+    for cdi_id in cdi_ids_input:
+       
+        idx = np.argmax(cdi_ids == cdi_id)
+        idxs.append(idx)
+
+    # create a list with the var that contain the temperature, salinity and depth values
+    var_names = [
+        name
+        for name, var
+        in ds.variables.items()
+        if (name.startswith('var') and not '_' in name) 
+    ]
+
+    # prepare the output
+    titles = ["Water temperature", "Water body salinity", "Depth" , "cdi_id"]
+    output = []
+    output.append(titles)
+
+    for idx in idxs:
+       
+        cdi_id = netCDF4.chartostring(ds.variables[cdi_id_var][idx])
+        
+        np.array2string(cdi_id)
+        
+       
+        
+        idx_variables ={} 
+        for var_name in var_names:
+            var = ds.variables[var_name]
+            try:
+                idx_variables[var.long_name] = var[idx]
+            except IdexError: 
+                print ("failed to index {} with index {}".format(var,  idx))
+        cdi_id_array = np.empty(shape = idx_variables["Depth"].shape, dtype = '<U28')
+        cdi_id_array.fill(str(cdi_id))
+
+        c= np.array(list(zip(idx_variables["ITS-90 water temperature"],idx_variables["Water body salinity"],idx_variables["Depth"])))
```

Comment:
We need to take a look at these hard coded var names.  Either be an inptu in the function, or something more generic .